### DISCUSSION TITLE
help 'C:\\Windows\\SYSTEM32\\VCRUNTIME140.dll' 14.0 is not compatible with this PHP build linked with 14.29 in Unknown on line 0

### DISCUSSION THREAD

**Discussion by anhcodervn** [leokhoa/laragon#579] [General]
Has anyone ever encountered this error? Can I ask how to fix it?

![image](https://github.com/leokhoa/laragon/assets/124981773/fec72338-67e4-4a54-9849-891985d6839d)

**Comment by anhcodervn** [comment]
PHP Warning:  'C:\\Windows\\SYSTEM32\\VCRUNTIME140.dll' 14.0 is not compatible with this PHP build linked with 14.29 in Unknown on line 0

**Comment by Pen-y-Fan** [comment]
What version of PHP are you trying to run?

You need a Visual Studio C++ run time compatible with the VC PHP was compiled with, see my blog [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/#selecting-the-correct-version), which explains in more details. 

The best place to download the latest redistributable is [Apache lounge](https://www.apachelounge.com/download/), see the link to "vc_redist_x64, see Redistributable"
